docs/content/concepts/object-ownership/party.mdx

```diff
@@ -35,4 +35,12 @@ Party objects whose owning address corresponds to an object ID are not supported
 
 ## When to use party objects
 
-Use party objects when you want an object to be versioned by consensus, such as for operational convenience. If an object is only used with other party or shared objects, converting it to a party object has no additional performance cost.
\ No newline at end of file
+Use party objects when you want an object to be versioned by consensus, such as for operational convenience. If an object is only used with other party or shared objects, converting it to a party object has no additional performance cost.
+
+Party objects can be used by multiple inflight transactions at the same time. This contrasts with address-owned objects, which only allow a single inflight transaction. Many applications may benefit from the ability to pipeline multiple transactions on the same party object.
+
+:::info
+
+`Coin`s can be party objects, including `Coin<SUI>`. However, it is not possible to use a party object `Coin<SUI>` for gas payment. If you want to use a party object `Coin<SUI>` for gas, it must first be transferred back to address-owned.
```

Comment:
```suggestion
`Coin` objects can be party-owned, including `Coin<SUI>`. However, you can't use a party-owned `Coin<SUI>` object for gas payment. To use a party-owned `Coin<SUI>` object for gas, you must first transfer it back to address-owned.
```

Reply:
from https://docs.sui.io/concepts/object-ownership - "address-owned" is the only ownership type with "-owned" in its name, the others are just X object, like party object, shared object, immutable object, etc.